diamond/solvers/diamond_logistic.py

```diff
@@ -69,8 +69,11 @@ def fit(Y, designs, H_invs, sparse_inv_covs, **kwargs):
 
             change = 0.0
             for grouping in fit_order:
-                g_change = np.linalg.norm(effects[grouping] - old_effects[grouping]) / \
-                           np.linalg.norm(effects[grouping])
+                if i > 0:
+                    g_change = np.linalg.norm(effects[grouping] - old_effects[grouping]) / \
+                               np.linalg.norm(effects[grouping])
+                else:
+                    g_change = np.inf
```

Comment:
is this to initialize the change in gradient for the first iteration? were we running into weird stopping behavior before?